client/src/SubscriptionDetailsCard.css

```diff
@@ -0,0 +1,74 @@
+.container {
+    backdrop-filter: blur(15px);
+    background: linear-gradient(180deg, #FFE2DF 0%, rgba(255, 226, 223, 0.39) 100%);
+    border: solid rgba(254, 229, 0, 1) 1px;
+    border-radius: 20px 20px 0 0;
+    display: flex;
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+    flex-direction: column;
+    height: 70vh;
+    padding: 20px 20px 0 20px;
+    width: 20vw;
+}
+
+.hoveredStyling {
+    border: 1px solid #FFFFFF;
+    box-shadow: inset 1px 0 20px 5px white;
+}
+
+.selectedStyling {
+    border: 1px solid #FFFFFF;
+    box-shadow: inset 1px 0 70px 20px white;
+}
```

Comment:
slightly funny to have styling in the name of the class, can we rename these classes to not have 'styling'?

Reply:
Agreed, I'll change it to omit 'styling' 👍